### PR TITLE
[Docs] Minor fixes for Elasticsearch documentation

### PR DESCRIPTION
**Pull request by bzz**
### What is this PR for?
Adds elasticsearch to interpreter menu in navigation and fixes max image size for screenshots

### What type of PR is it?
Documentation | Hot Fix

### How should this be tested?
Build docs, go to elasticsearch one.

### CODE REVIEW

**Comment by bzz** [comment]
This is HotFix for #520 
Would like to merge it asap

File: docs/_includes/themes/zeppelin/_navigation.html [hunk 1]
```diff
@@ -47,6 +47,7 @@
                 <li><a href="{{BASE_PATH}}/pleasecontribute.html">Shell</a></li>
                 <li><a href="{{BASE_PATH}}/interpreter/spark.html">Spark</a></li>
                 <li><a href="{{BASE_PATH}}/pleasecontribute.html">Tajo</a></li>
+                <li><a href="{{BASE_PATH}}/interpreter/elasticsearch.html">Elasticsearch</a></li>
```
Comment (Leemoonsoo) [inline docs/_includes/themes/zeppelin/_navigation.html]: Can we keep alphabetical order?

Comment (bzz) [inline docs/_includes/themes/zeppelin/_navigation.html]: Sure!

**Comment by bzz** [comment]
\cc @bbonnin for review

**Comment by bzz** [comment]
Ready to be merged

**Comment by bbonnin** [comment]
Looks good to me.

**Comment by Leemoonsoo** [comment]
LGTM

**Comment by bzz** [comment]
Thank you for fast reviews!

Published live on http://zeppelin.incubator.apache.org/docs/0.6.0-incubating-SNAPSHOT/interpreter/elasticsearch.html

**Pull request closed without merging**